addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -9098,12 +9104,12 @@ QUnit.module("Views", (hooks) => {
         };
         await click(target.querySelector("button.p"));
         // edit the new row again and set a many2one value
-        await clickLast(
-            target.querySelector(".o_form_view .o_field_one2many .o_data_row .o_data_cell")
+        await click(
+            target.querySelectorAll(".o_form_view .o_field_one2many .o_data_row .o_data_cell")[1]
         );
-        await testUtils.nextTick();
-        await testUtils.fields.many2one.clickOpenDropdown("product_id");
-        await testUtils.fields.many2one.clickHighlightedItem("product_id");
+        await nextTick();
+        await click(target.querySelector(".o-autocomplete--input"));
+        await click(target.querySelector(".dropdown-item"));
```

Comment:
I think you can replace these two lines by `await selectDropdownItem(target, "product_id", "xphone");`